scan/cli.go

```diff
@@ -230,13 +252,38 @@ func createGenericAuditCmd(c *cli.Context) (*audit.GenericAuditCommand, error) {
 	if c.String("working-dirs") != "" {
 		auditCmd.SetWorkingDirs(splitAndTrim(c.String("working-dirs"), ","))
 	}
+	auditCmd.GraphBasicParams = &xrCmdUtils.GraphBasicParams{}
```

Comment:
It's better to initialize this struct inside the `NewGenericAuditCommand()` function.